CHANGELOG.md

```diff
@@ -1,3 +1,67 @@
+* remove deprecated modules ([#2693](https://github.com/awslabs/aws-cdk/issues/2693)) ([421bf6d](https://github.com/awslabs/aws-cdk/commit/421bf6d))
+* **core:** improvements to Construct API ([#2767](https://github.com/awslabs/aws-cdk/issues/2767)) ([3f7a0ad](https://github.com/awslabs/aws-cdk/commit/3f7a0ad)), closes [#1934](https://github.com/awslabs/aws-cdk/issues/1934) [#2766](https://github.com/awslabs/aws-cdk/issues/2766)
+* **core:** misc cleanups to App-related APIs ([#2731](https://github.com/awslabs/aws-cdk/issues/2731)) ([b2e1964](https://github.com/awslabs/aws-cdk/commit/b2e1964)), closes [#1891](https://github.com/awslabs/aws-cdk/issues/1891) [#2750](https://github.com/awslabs/aws-cdk/issues/2750)
+* **kms:** Rename EncryptionKeyAlias to Alias ([#2769](https://github.com/awslabs/aws-cdk/issues/2769)) ([da8e1d5](https://github.com/awslabs/aws-cdk/commit/da8e1d5))
+
+
+### Features
+
+* **core:** node.defaultChild as a shortcut to escape hatch ([#2684](https://github.com/awslabs/aws-cdk/issues/2684)) ([8cd3c23](https://github.com/awslabs/aws-cdk/commit/8cd3c23)), closes [#2290](https://github.com/awslabs/aws-cdk/issues/2290)
+* **core+cli:** support tagging of stacks ([#2185](https://github.com/awslabs/aws-cdk/issues/2185)) ([d0e19d5](https://github.com/awslabs/aws-cdk/commit/d0e19d5)), closes [#932](https://github.com/awslabs/aws-cdk/issues/932)
+* **event-targets:** add support for fargate/awsvpc tasks ([#2707](https://github.com/awslabs/aws-cdk/issues/2707)) ([2754dde](https://github.com/awslabs/aws-cdk/commit/2754dde))
+* **events:** support SQS queues as rule targets ([#2683](https://github.com/awslabs/aws-cdk/issues/2683)) ([078e34a](https://github.com/awslabs/aws-cdk/commit/078e34a)), closes [#1786](https://github.com/awslabs/aws-cdk/issues/1786)
+* **rds:** add engineVersion to DatabaseCluster ([#2698](https://github.com/awslabs/aws-cdk/issues/2698)) ([0593d51](https://github.com/awslabs/aws-cdk/commit/0593d51)), closes [#2212](https://github.com/awslabs/aws-cdk/issues/2212)
+* **rds:** add support for database instances ([#2187](https://github.com/awslabs/aws-cdk/issues/2187)) ([b864041](https://github.com/awslabs/aws-cdk/commit/b864041)), closes [#2075](https://github.com/awslabs/aws-cdk/issues/2075) [#1693](https://github.com/awslabs/aws-cdk/issues/1693)
+* **route53:** improve constructs for basic records ([#2741](https://github.com/awslabs/aws-cdk/issues/2741)) ([696f53f](https://github.com/awslabs/aws-cdk/commit/696f53f))
+* **s3:** default to KMS if encryptionKey is specified ([#2719](https://github.com/awslabs/aws-cdk/issues/2719)) ([ae4a04f](https://github.com/awslabs/aws-cdk/commit/ae4a04f)), closes [#2714](https://github.com/awslabs/aws-cdk/issues/2714)
+* **tokens:** enable type coercion ([#2680](https://github.com/awslabs/aws-cdk/issues/2680)) ([0f54698](https://github.com/awslabs/aws-cdk/commit/0f54698)), closes [#2679](https://github.com/awslabs/aws-cdk/issues/2679)
+
+
+### BREAKING CHANGES
```

Comment:
The following breaking changes from #2187 are missing (were not included in the final PR description):
* `rds.RotationSingleUser` renamed to `rds.SecretRotation`
* `rds.ClusterParameterGroup` no longer have `setParameter` and `removeParameter` methods, use the `parameters` prop directly in the constructor instead.